apps/dotcom/client/src/tla/providers/TlaAppProvider.tsx

```diff
@@ -1,11 +1,24 @@
+import { ClerkProvider } from '@clerk/clerk-react'
 import { Outlet } from 'react-router-dom'
 import { AppStateProvider } from '../hooks/useAppState'
+import { UserProvider } from '../hooks/useUser'
 import '../styles/tla.css'
 
+// @ts-ignore this is fine
+const PUBLISHABLE_KEY = import.meta.env.VITE_CLERK_PUBLISHABLE_KEY
+
 export function Component() {
+	if (!PUBLISHABLE_KEY) {
+		throw new Error('Missing Publishable Key')
+	}
+
 	return (
 		<AppStateProvider>
-			<Outlet />
+			<ClerkProvider publishableKey={PUBLISHABLE_KEY} afterSignOutUrl="/">
```

Comment:
when merging with my stuff I think we'll need to hoist this so it can be available on the logged out page and if the user is logged in, redirect them.
```suggestion
			<ClerkProvider publishableKey={PUBLISHABLE_KEY} afterSignOutUrl="/q">
```